templates/k-bims-ledet/0/docker-compose.yml

```diff
@@ -0,0 +1,362 @@
+      UWSGI_CMD: uwsgi --ini /usr/src/ledet/uwsgi.conf
+      GEONODE_GEODATABASE: geonode_data
+      GEONODE_DATABASE_PASSWORD: docker
+      C_FORCE_ROOT: 1
+    entrypoint: /bin/sh
+    command: -c "python manage.py collectstatic --noinput -i geoexplorer"
+    volumes:
+    - django-static:/home/web/static:rw
+    - django-media:/home/web/media:rw
+    - django-reports:/home/web/reports
+    - django-logs:/var/log/
+    links:
+      - geoserver:geoserver
+      - rabbitmq:rabbitmq
+      - db:db
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: true
+    restart: unless-stopped
+
+  migrate:
+    # This container will just run, migrations
+    # and then exit. Restart it again in rancher to
+    # rerun migrations.
+    # Inherit from the uwsgi container
+    # We need to migrate accounts (auth) first as it has a reference to user model
+    image: kartoza/ledet_uwsgi:latest
+    environment:
+      DATABASE_HOST: db
+      DATABASE_NAME: gis
+      DATABASE_PASSWORD: docker
+      DATABASE_USERNAME: docker
+      DJANGO_SETTINGS_MODULE: ledet_core.settings.prod_docker
+      RABBITMQ_HOST: rabbitmq
+      VIRTUAL_HOST: healthyrivers.kartoza.com
+      VIRTUAL_PORT: '8080'
+      SITE_DOMAIN_NAME: http://${SITE_URL}
+      SITEURL: http://${SITE_URL}/
+      PYTHONPATH: /usr/src/app:/usr/src/bims:/home/web/django_project
+      HAYSTACK_HOST: elasticsearch
+      GEOCONTEXT_URL: ${GEOCONTEXT_URL}
+      CONTACT_US_EMAIL: ${ADMIN_EMAIL}
+      ROOT_URLCONF: ledet_core.urls
+      GEOSERVER_PUBLIC_LOCATION: http://${SITE_URL}/geoserver/
+      STATIC_ROOT: /home/web/static
+      GEONODE_DATABASE: gis
+      GEONODE_DATABASE_USER: docker
+      MEDIA_ROOT: /home/web/media
+      ADMIN_EMAILS: ${ADMIN_EMAIL}
+      GEOSERVER_ADMIN_PASSWORD: ${GEOSERVER_ADMIN_PASSWORD}
+      GEOIP_PATH: /home/web/media/geoip.db
+      GEONODE_GEODATABASE_PASSWORD: geonode_data
+      GEONODE_INSTANCE_NAME: geonode
+      DEFAULT_BACKEND_DATASTORE: datastore
+      IS_CELERY: 'False'
+      ALLOWED_HOSTS: '["localhost","127.0.0.1","${SITE_URL}"]'
+      GEOSERVER_LOCATION: http://geoserver:8080/geoserver/
+      UWSGI_CMD: uwsgi --ini /usr/src/ledet/uwsgi.conf
+      GEONODE_GEODATABASE: geonode_data
+      GEONODE_DATABASE_PASSWORD: docker
+      C_FORCE_ROOT: 1
+    entrypoint: /bin/sh
+    command: -c "python manage.py makemigrations --noinput;python manage.py migrate auth;python manage.py migrate"
+    volumes:
+    - django-static:/home/web/static:rw
+    - django-media:/home/web/media:rw
+    - django-reports:/home/web/reports
+    - django-logs:/var/log/
+    links:
+      - geoserver:geoserver
+      - rabbitmq:rabbitmq
+      - db:db
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: true
+    restart: unless-stopped
+
+  rabbitmq:
+    image: library/rabbitmq
+    environment:
+      RABBITMQ_NODENAME: rabbit
+      RABBIT_PASSWORD: rabbit_test_password
```

Comment:
Another candidate for rancher conf?

Reply:
Maybe I should just delete that instead. I'm not sure who uses rabbitmq in this case (I copied it from generic bims).
But, I don't think we need rancher conf from this since we can config it from the environment vars.

Reply:
@timlinux apparently @dimasciput wants to use rabbitmq. I will prepare more proper fix in the next PR.